### ISSUE TITLE
Regarding the Standard JS logo consistency in badges

### ISSUE THREAD

**Issue by revelt** [standard/standard-www#15]
Hi all,

The official JS Standard badge, in black and white, 

[![JavaScript Style Guide](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://github.com/feross/standard)

is not very popular as far as I can see and many people use the sticker SVG instead:

<a href="https://standardjs.com"><img src="https://cdn.rawgit.com/feross/standard/master/sticker.svg" alt="Standard JavaScript" width="100"></a>

I believe that [is-there-a-readme-badge](https://github.com/feross/standard#is-there-a-readme-badge) section should be supplemented with a hexagonal yellow sticker logo and it should be made the official badge.

Secondly, I believe that the badge is better to be floated to the top right corner of the README, so it takes less space. My favourite Markdown snippet for that is this:

```html
<a href="https://standardjs.com" style="float: right; padding: 0 0 20px 20px;"><img src="https://cdn.rawgit.com/feross/standard/master/sticker.svg" alt="Standard JavaScript" width="100" align="right"></a>
```

It adds some margin and it even [works](https://github.com/npm/marky-markdown/issues/269) on npm readme's. I personally chased npm and made them allow badge floating 😃 

Now, according to the seminal book "The 22 Immutable Laws of Branding", the rule No.19, "The Law of Consistency" it would be a good idea to drop the black and white badge _completely_, and stick to a **consistent** branding — a yellow hexagonal badge everywhere: on badges, on real stickers, on the https://standardjs.com and everywhere else.

### One logo and only one, used everywhere.

Please share your opinions.

**Comment by jprichardson** [comment]
I agree with this and think it's a wonderful idea. It's much more recognizable. 

To see it in action: https://github.com/jprichardson/node-klaw/blob/master/README.md

**Comment by Flet** [comment]
I'm still fond of the ol' black and white logo, but your reasoning makes a lot of sense :)

It might be best to open an issue/PR on [feross/standard](https://github.com/feross/standard) to get a bit more visibility and feedback.

The standardjs.com site will be updated automatically within a few minutes once the feross/standard README us updated.

**Comment by bcomnes** [comment]
I like it, though not everyone is going to want the big yellow logo at the top of their readme.  Another thing I see is people using shields.io to denote standard code style.  We should maybe add it to the badge section though?   I wonder if putting the yellow hexagon into the standard badge would look cool too.

**Comment by LinusU** [comment]
I personally like the black and white one very much ❤️
